### PR TITLE
Update code and some dependecies

### PR DESCRIPTION
**Pull request by caiosantanaj**
I split the Game component and put some new features of React ([Hooks](https://reactjs.org/docs/hooks-intro.html))

### CODE REVIEW

File: src/Components/Game.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  return data === null ? (
+    <h1>Loading </h1>
+  ) : (
+    <div>
+      <Question id={currentQuestionId} question={data.data} />
+      <Options
+        id={currentQuestionId}
+        options={data.data}
+        updateQuestionId={updateQuestionId}
+      />
```
Comment (T-Almeida) [inline src/Components/Game.js:39]:
```suggestion
      <Question id={currentQuestionId} question={data.data[currentQuestionId].query} />
      <Options
        id={currentQuestionId}
        options={data.data[currentQuestionId].responses}
        updateQuestionId={updateQuestionId}
      />
```

**Review by T-Almeida — CHANGES_REQUESTED** [review]
Seems fine.

However, I would suggest the split of the data at the Game level, following the Components logic.

The snippet should account for these changes, just confirm if it is indeed correct.

File: src/Components/Game.js (hunk 1 above)
Comment (T-Almeida) [reply src/Components/Game.js:39]: Ah it is also required the respective change on the Question and Options Component

Comment (caiosantanaj) [reply src/Components/Game.js:39]: Ok!  :)

**Pull request merged**